daemon/list.go

```diff
@@ -303,6 +303,24 @@ func (daemon *Daemon) foldFilter(view container.View, config *types.ContainerLis
 
 	err = psFilters.WalkValues("before", func(value string) error {
 		beforeContFilter, err = view.Get(value)
+		switch err.(type) {
```

Comment:
It looks like both of these `func(value string) error` are identical, so it should probably be a function called in both places to return the filter.

Reply:
Thanks @dnephin, the PR has been updated.